python/kfserving/README.md

```diff
@@ -1 +1,39 @@
-TODO
\ No newline at end of file
+# KFServing
+
+KFServing is a unit of model serving, and every model type (XGBosst, PyTorch, Tensorflow etc.) extends KFServing. At a highlevel it defines a model, a server and storage mechanism for models.
```

Comment:
It may be good to differentiate the python and golang kfservings.

"KFServing's python libraries implement a standardized KFServer library that is extended by model serving frameworks such as XGBoost and PyTorch. It encapsulates data plane API definitions and storage retrieval for models.

Reply:
Done